cilium/cmd/kvstore_delete.go

```diff
@@ -27,18 +28,21 @@ var kvstoreDeleteCmd = &cobra.Command{
 	Short:   "Delete a key",
 	Example: "cilium kvstore delete --recursive foo",
 	Run: func(cmd *cobra.Command, args []string) {
-		setupKvstore()
-
 		if len(args) < 1 {
 			Fatalf("Please specify a key or key prefix to delete")
 		}
 
+		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
Should we add a default timeout const variable for the `30*time.Second` for all of this CLI?

Reply:
@ianvernon please follow-up with @joestringer's feedback above in a separate PR, thanks.